cosmovisor/README.md

```diff
@@ -322,18 +313,13 @@ mkdir -p $DAEMON_HOME/cosmovisor/upgrades/test1/bin
 cp ./build/simd $DAEMON_HOME/cosmovisor/upgrades/test1/bin
 ```
 
-Start `cosmosvisor`:
-
-```
-cosmovisor run start
-```
```

Comment:
I think we should start cosmovisor here and use a block height of 20. This allowed for a smoother user experience. The instructions currently do not give users enough time to implement the upgrade handler before block height 100 and they will likely need to start over.

Also, it might make sense to create a section following "Update App" called "Run Example" that starts here with running cosmovisor.

Reply:
Moved. 
I will change block height to 200 and add a comment. We need to run cosmovisor before "Update App" and after chain setup.

Reply:
I was suggesting block height 20 and starting cosmovisor after update app. It is not necessary to run cosmovisor before update app (see [Chain Upgrade Guide](https://docs.cosmos.network/v0.44/migrations/chain-upgrade-guide-044.html)). Ok to leave as is though.